mars-sim-core/src/main/java/org/mars_sim/msp/core/person/ai/social/Relation.java

```diff
@@ -21,18 +21,24 @@
  */
 public class Relation implements Serializable {
 
+	/**
+	 * Details of a specific relationship to a Person
+	 */
+	public record Opinion(double trust, double care, double respect) implements Serializable {
```

Comment:
Great. We're now using record for the very first time in mars-sim.

I suppose when we switch to JSON serialization, record would be supported, right ?